python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -0,0 +1,405 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import lsst.afw.detection as afwDetect
+import lsst.afw.table as afwTable
+import lsst.meas.base as measBase
+import lsst.daf.base as dafBase
+import lsst.pipe.base as pipeBase
+import lsst.pex.config as pexConfig
+from lsst.meas.base import MeasurementError
+from lsst.pipe.tasks.characterizeImage import CharacterizeImageTask
+from lsst.meas.algorithms.installGaussianPsf import InstallGaussianPsfTask
+
+
+class QuickFrameMeasurementTaskConfig(pexConfig.Config):
+    """Config class for the QuickFrameMeasurementTask."""
+    maxNonRoundness = pexConfig.Field(
+        dtype=float,
+        doc="Ratio of xx to yy (or vice versa) above which to cut, in order to exclude spectra",
+        default=5.,
+    )
+    maxExtendedness = pexConfig.Field(
+        dtype=float,
+        doc="Max absolute value of xx and yy above which to cut, in order to exclude large/things",
+        default=100,
+    )
+    doExtendednessCut = pexConfig.Field(
+        dtype=bool,
+        doc="Apply the extendeness cut, as definted by maxExtendedness",
+        default=False,
+    )
+    sanityCheckPercentile = pexConfig.Field(
```

Comment:
Sure. I went with `doCheckCentroidPixelValue` and `centroidPixelPercentile` because we don't actually enforce it being a `peak` in the technical term-of-art sense.